packages/devtools_app/test/integration_tests/integration.dart

```diff
@@ -58,13 +56,14 @@ class BrowserManager {
   BrowserManager._(this.chromeProcess, this.tab);
 
   static Future<BrowserManager> create() async {
-    final Chrome chrome = Chrome.locate();
+    final Chrome? chrome = Chrome.locate();
     if (chrome == null) {
       throw 'unable to locate Chrome';
     }
 
     final ChromeProcess chromeProcess = await chrome.start();
-    final ChromeTab tab = await chromeProcess.getFirstTab();
+    final ChromeTab tab =
+        await (chromeProcess.getFirstTab() as FutureOr<ChromeTab>);
```

Comment:
couple things - 1) why do we need the cast, and 2) `getFirstTab` returns `Future<ChromeTab?>`, so doesn't this need to be nullable or null-asserted? same question below.